packages/grafana-runtime/src/services/AngularLoader.ts

```diff
@@ -1,19 +1,81 @@
+/**
+ * Used to enable rendering of angular components within a
+ * react component without loosing proper typings.
+ *
+ * @example
+ * ```typescript
+ * class Component extends PureComponent<Props> {
+ *   element: HTMLElement;
+ *   angularComponent: AngularComponent;
+ *
+ *   componentDidMount() {
+ *     const template = '' // angular template here;
+ *     const scopeProps = {}; // angular scope properties here
+ *     const loader = getAngularLoader();
+ *     this.angularComponent = loader.load(this.element, scopeProps, template);
+ *   }
+ *
+ *   componentWillUnmount() {
+ *     if (this.angularComponent) {
+ *       this.angularComponent.destroy();
+ *     }
+ *   }
+ *
+ *   render() {
+ *     return (
+ *       <div ref={element => (this.element = element)} />
+ *     );
+ *   }
+ * }
+ * ```
+ *
+ * @public
+ */
```

Comment:
```suggestion
/**
 * Used to enable rendering of Angular components within a
 * React component without loosing proper typings.
 *
 * @example
 * ```typescript
 * class Component extends PureComponent<Props> {
 *   element: HTMLElement;
 *   angularComponent: AngularComponent;
 *
 *   componentDidMount() {
 *     const template = '' // Angular template here;
 *     const scopeProps = {}; // Angular scope properties here
 *     const loader = getAngularLoader();
 *     this.angularComponent = loader.load(this.element, scopeProps, template);
 *   }
 *
 *   componentWillUnmount() {
 *     if (this.angularComponent) {
 *       this.angularComponent.destroy();
 *     }
 *   }
 *
 *   render() {
 *     return (
 *       <div ref={element => (this.element = element)} />
 *     );
 *   }
 * }
 * ```
 *
 * @public
 */
```